accessibility.md

```diff
@@ -0,0 +1,66 @@
+# Accessibility
+
+## Compliance
+
+We use a subset of the WCAG 2.1 AA standards as our benchmark for accessible products.
+
+Aim to build user interfaces that:
+
+- Provide [text alternatives](https://www.w3.org/TR/WCAG21/#text-alternatives)
+- Are [adaptable](https://www.w3.org/TR/WCAG21/#adaptable)
+- Are [distinguishable](https://www.w3.org/TR/WCAG21/#distinguishable)
+- Do not [trigger seizures or physical reactions](https://www.w3.org/TR/WCAG21/#seizures-and-physical-reactions)
+- Are [navigable](https://www.w3.org/TR/WCAG21/#navigable)
+- Are [predictable](https://www.w3.org/TR/WCAG21/#navigable)
+- Help users [avoid or correct mistakes](https://www.w3.org/TR/WCAG21/#input-assistance)
+- Are [compatible](https://www.w3.org/TR/WCAG21/#compatible) with existing or future users agents and assistive technologies
```

Comment:
I am a big fan of the [reference-style link notation][md] in Markdown. It keeps a better flow for longer sentences.

[md]: https://daringfireball.net/projects/markdown/syntax#link "I’m using it here, too!"

```suggestion
- Are [compatible][] with existing or future users agents and assistive technologies

[compatible]: (https://www.w3.org/TR/WCAG21/#compatible)
```

Reply:
Cool! Will the output look any different?

Reply:
Not at all!